doc/first-steps.md

```diff
@@ -1,70 +1,27 @@
-# getting started using the images
+# Getting started using the images
```

Comment:
quite a few things are being removed here like the keyboard language setup, the initial setup script - is there some replacement for that in the pipeline already (or maybe i have overlooked it somewhere else?) ... in general i would prefer to remove parts only if there is a replacement for them in place already

Reply:
yeah i forgot to bring these back

Reply:
https://github.com/hexdump0815/imagebuilder/pull/255/commits/1296acf9e79e5fe9908f9fe7cef0d0a0b096c4b8

should do the trick
if there is anything else missing let me know